### PR TITLE
Qianwen/tag-0.9.30

### PR DESCRIPTION
**Pull request by wangqianwen0418**

### CODE REVIEW

**Comment by wangqianwen0418** [comment]
@sehilyi, when you are convenient, could you take a look and see whether there are other updates we should include about v0.9.30?

File: versioned_docs/version-0.9.30/FAQ.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+<img src="https://user-images.githubusercontent.com/44389194/136196719-abefef53-49e0-4ea5-8dae-a438fa6d5f33.png" alt="bug_screenshot"/>
+
+For problems like this, please ensure that you have added the following style sheets in your base html file.
+
+```html
+<head>
+  ...
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+  <link rel="stylesheet" href="https://unpkg.com/higlass@1.11.3/dist/hglib.css">
```
Comment (sehilyi) [inline versioned_docs/version-0.9.30/FAQ.md:35]: Given that the latest version of Gosling.js is compatible with HiGlass v1.12.4, let's update this.

```suggestion
  <link rel="stylesheet" href="https://unpkg.com/higlass@1.12.4/dist/hglib.css">
```

File: versioned_docs/version-0.9.30/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+```html
+<!DOCTYPE html>
+<html>
+<head>
+  <link rel="stylesheet" href="https://unpkg.com/higlass@1.11/dist/hglib.css">
+  <script src="https://unpkg.com/react@17/umd/react.production.min.js"></script>
+  <script src="https://unpkg.com/react-dom@17/umd/react-dom.production.min.js"></script>
+  <script src="https://unpkg.com/pixi.js@6/dist/browser/pixi.min.js"></script>
+  <script src="https://unpkg.com/higlass@1.11/dist/hglib.js"></script>
+  <script src="https://unpkg.com/gosling.js@0.9.17/dist/gosling.js"></script>
```
Comment (sehilyi) [inline versioned_docs/version-0.9.30/getting-started.md:33]:
```suggestion
  <link rel="stylesheet" href="https://unpkg.com/higlass@1.12.4/dist/hglib.css">
  <script src="https://unpkg.com/react@17/umd/react.production.min.js"></script>
  <script src="https://unpkg.com/react-dom@17/umd/react-dom.production.min.js"></script>
  <script src="https://unpkg.com/pixi.js@6/dist/browser/pixi.min.js"></script>
  <script src="https://unpkg.com/higlass@1.12.4/dist/hglib.js"></script>
  <script src="https://unpkg.com/gosling.js@0.9.32/dist/gosling.js"></script>
```

File: versioned_docs/version-0.9.30/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+Install `gosling.js` and its dependent libraries:
+
+```sh
+yarn add gosling.js higlass pixi.js react@16.13.1 react-dom@16.13.1
+```
+
+Add the following stylesheet to your base `html` file:
+```html
+<head>
+  <link rel="stylesheet" href="https://unpkg.com/higlass@1.11.3/dist/hglib.css">
```
Comment (sehilyi) [inline versioned_docs/version-0.9.30/getting-started.md:79]:
```suggestion
  <link rel="stylesheet" href="https://unpkg.com/higlass@1.12.4/dist/hglib.css">
```

**Review by sehilyi — COMMENTED** [review]
I made a small number of suggestions mainly related to updating the version of higlass/gosling.js in the code examples. Thank you for updating the documentation!!

**Comment by wangqianwen0418** [comment]
Thanks Sehi!

**Pull request merged**